### ISSUE TITLE
[Supporting the platform - ADF - Azure data factory] On the Azure data factory window, the clear completed button doesn't adapt high contrast themes. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3872]
**User Experience:**
If the button is not adapting the high contrast theme then the visually impaired users for navigating might get affected hence failed to get the proper information of the button.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it. 
System preferences: Invert colors

**Repro Steps:​**
1. Launch Azure Storage Explorer app.
2. Navigate and select visual icon control Add data factory control at the left panel.
3. Navigate with the keyboard and reach "Clear completed" control on the Azure data factory window.
4. Apply a high contrast theme on the azure factory window.

**Actual Result:**
On the Azure data factory window, the clear completed button doesn't adapt high contrast themes i.e Clear completed button appears like a link.

**Expected Result:** 
On the Azure data factory window, the clear completed button should adapt the high contrast themes i.e Clear completed button appears like a button, not a link.

**Note:**
This issue also exists for clear successful button.
![MAS4 3 1-Clear completed button does not adapting high contrast theme](https://user-images.githubusercontent.com/72871089/101479311-ff2d9180-3977-11eb-8525-e54bd3cbecb2.png)

**Comment by MRayermannMSFT** [comment]
This issue was fixed some time ago.
